org-code-javabuilder/theater/src/main/java/org/code/theater/Stage.java

```diff
@@ -0,0 +1,210 @@
+package org.code.theater;
+
+public enum Instrument {
+   PIANO, BASS
+}
+
+public class Stage {
+   /**
+    * Returns the width of the theater canvas. Right now this will always be 400
+    * pixels.
+    */
+   public double getWidth() {
```

Comment:
Will we be working in fractional pixels?  Wondering if width and height should be ints or doubles?  How about x and y?  (This applies to all APIs in this class.)

Reply:
AP CSA only teaches `double` as the primitive numeric type 🤷

Reply:
I see `int` and `Integer` referenced in the AP CSA Java subset?  https://secure-media.collegeboard.org/digitalServices/pdf/ap/ap-computer-science-a-java-subset.pdf

Reply:
Oh you're right, I had been thinking floats. Yes we should move these to `int`.

Reply:
@molly-moen can you move the getWidth/getHeight methods to return ints? 

All other methods that refer to x & y should remain doubles so you can draw shapes at fractional positions.